examples/identity-app/README.md

```diff
@@ -26,19 +26,46 @@ Be sure that Docker image von-network is on your Docker registry.
 If you have not the Docker image on your Docker registry, then build them.
 Please follow [Documentation](./images/von-network/README.md)
 ### Step 2
-Prepare a network.yaml file for your demo or you can use this sample for [AWS](../../platforms/hyperledger-indy/configuration/samples/network-indyv3-aries.yaml) or [Minikube](../../platforms/hyperledger-indy/configuration/samples/network-minikube-aries.yaml) and update for your case.
+Be sure that Docker image of agents is on your Docker registry.
+If you have not the Docker image on your Docker registry, then build them.
+Please follow [Documentation](./images/agents/README.md) 
 ### Step 3
+Prepare a network.yaml file for your demo or you can use this sample for [AWS](../../platforms/hyperledger-indy/configuration/samples/network-indyv3-aries.yaml) or [Minikube](../../platforms/hyperledger-indy/configuration/samples/network-minikube-aries.yaml) and update for your case.
+### Step 4
 Run Ansible Playbook by these [steps](../../platforms/hyperledger-indy/configuration/README.md)
 Can be started also via Docker container:
 1. Go to root directory of this git repository.
 2. Run command: `docker run -it -v $(pwd):/home/blockchain-automation-framework/ --entrypoint bash hyperledgerlabs/baf-build`
 3. When are you entered into Docker container console, use command:
 `./run.sh`
-### Step 4
-After Blockchain Automation Framework setup can be created Helm release of Indy Web Server:
-1. Create Helm release with command: `helm install --name authority-webserver ./charts/webserver/`
+### Step 5
+After Blockchain Automation Framework setup can be deployed Identity App via Ansible playbook:
+
+- Be sure that you are in Docker container (BAF), if not, then enter via command: `docker exec -it <docker_container_name_or_id> bash`
+- Update Trustee service in your `network.yaml` file to add port of Indy WebServer, which will run with trustee role.
+example of trustee: 
+```yaml
+      services:
+        trustees:
+        - trustee:
+          name: authority-trustee
+          genesis: true
+          server:
+            port: 8000
+```
+- Run Ansible [playbook](./configuration/deploy-identity-app.yaml) with command: `ansible-playbook -e "ansible_python_interpreter=/usr/local/bin/python3" -i ./blockchain-automation-framework/platforms/shared/inventory/ansible_provisoners ./blockchain-automation-framework/examples/identity-app/configuration/deploy-identity-app.yaml -e "@./blockchain-automation-framework/build/network.yaml"`
+
+### Step 6
+Agents for University Faber and Student Alice have no Ansible roles created yet. These agents have to be run manualy:
```

Comment:
"Agents for Faber University and Student Alice don't have Ansible roles created yet. These agents have to be run manually"